src/RuleSet/Sets/PERCS10RiskySet.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\RuleSet\Sets;
+
+use PhpCsFixer\RuleSet\AbstractRuleSetDescription;
+
+/**
+ * @internal
+ *
+ * PER Coding Style v1.0.
+ *
+ * @link https://github.com/php-fig/per-coding-style/blob/1.0.0/spec.md
+ */
+final class PERCS10RiskySet extends AbstractRuleSetDescription
+{
+    public function getName(): string
```

Comment:
I get the idea behind overriding this method, but I think it shows the problem we are bypassing here.
looking at class name, we still do now know if it's PERCS 1.0 or 10. I wonder if we can adjust class name too?

Reply:
I am open to suggestions, given that we can't have periods in class names.  Using getName() is what @julienfalque suggested.  I have no particular preference, so whatever works.

Reply:
`PERCS1x0RiskySet`? and `AbstractRuleSetDescription` can replace `\dx\d` with `\1.\2`

Reply:
Let's use hardcoded values - we can easily change it in the future without breaking change, because name won't change, even if it's calculated automagically from class name.

PS. I removed that `Preg::replace()` which was added before at some point, because it wasn't used. If we actually need it, we can add it in separate PR.